### ISSUE TITLE
Essentials /ignore

### ISSUE THREAD

**Issue by LoXR** [EssentialsX/Essentials#1979]
### Information

Full output of `/ess version`: Essentials version 2.15.0.1

### Details

If not implented already, just wanting it so when a player does /ignore it /ignore messages aswell as the ignored players chat.

**Comment by ghost** [comment]
Can you rephrase? Don't understand what you're requesting here.

**Comment by LoXR** [comment]
For /ignore to not just ignore messages but to also ignore the ignored players chat messages in chat.

**Comment by ghost** [comment]
That's... exactly what /ignore does... don't show a specified user's chat messages.

**Comment by mdcfe** [comment]
@Dan9er `/ignore` prevents a player receiving private messages and mails from a user. ~~This is a valid feature request.~~

**Comment by LoXR** [comment]
Think its due to a bungee plugin BungeeMSG, its conflicting with Essentials ignore and still allowing messages to go through.

**Comment by mdcfe** [comment]
If `/ignore` still isn't working as expected, could you try using `/essentials:ignore`?

**Comment by LoXR** [comment]
I'll close this as its the plugin BungeeMSG, it hasnt been updated and overrides /ignore. May have to disable that feature so essentials can go its thing on servers.
